### ISSUE TITLE
Option to only show the "download" button on hover

### ISSUE THREAD

**Issue by daattali** [timelyportfolio/exportwidget#3]
The "download" button in the top right corner is not the prettiest looking and can really clash with the graphics of an output object. It could be nice to be able to only show it when the widget is hovered over.

**Comment by timelyportfolio** [comment]
I agree it is ugly.  Styling, positioning, and interaction can all be significantly improved.  Unfortunately, all my spare time over next couple of weeks will be consumed prepping for [PlotCon 2016](https://plotcon.plot.ly/), so I won't be able to work on it until then.

One concern I have is that since this is intended to be generic I worry that event handling on the export button might conflict with event handling on the widget. 

Some keystroke would be really nice, but not sure on a page with multiple how to choose which to export on a keystroke.  Maybe could export all with a keystroke.

**Comment by daattali** [comment]
I was thinking to simply have an option that would only show the button on hover. It might get complicated, but I don't imagine it having too much complications with being generic - the htmlwidget element can have a hover event listener added to it. Sounds simple to me, but I don't know how the code works so you'd know better than me.

If you want to be really fancy with keystrokes (along the lines of what you suggested, but not what I was thinking), you could have a keystroke that, when performed, looks at all the htmlwidgets on the page and gives each one an ID. Then the user can select an ID, or "select all". A-la chrome vim: ![chrome vim](http://usevim.com/images/posts/cvim.png) 

Just a crazy idea that I'm not actually expecting to happen....
